config/config/pom.xml

```diff
@@ -52,12 +52,7 @@
         </dependency>
         <dependency>
             <groupId>io.helidon.inject</groupId>
-            <artifactId>helidon-inject-api</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.helidon.inject</groupId>
-            <artifactId>helidon-inject-runtime</artifactId>
-            <optional>true</optional>
+            <artifactId>helidon-inject-service</artifactId>
```

Comment:
Because we do `provides ModuleComponet with Injection__Component` - and that cannot be done for a static dependency. This is a drawback of modularized applications. There is currently no workaround, except for creating an additional module